Modules/CTP/src/RawDataQcTask.cxx

```diff
@@ -115,18 +119,29 @@ void CTPRawDataReaderTask::startOfActivity(const Activity& activity)
     ILOG(Warning, Support) << "CTP config not found, run:" << run << ENDM;
   }
   if (mIndexMBclass == -1) {
+    MBclassName = "CMBV0 (default)";
     mIndexMBclass = 1;
   }
-  std::string nameInput1 = mCustomParameters.atOrDefaultValue("MB1inputName", "MTVX", activity);
-  std::string nameInput2 = mCustomParameters.atOrDefaultValue("MB2inputName", "MT0A", activity);
+  std::string nameInput1 = getFromExtendedConfig<string>(activity, mCustomParameters, "MB1inputName", "MTVX");
+  std::string nameInput2 = getFromExtendedConfig<string>(activity, mCustomParameters, "MB2inputName", "MT0A");
+
+  auto input2Tokens = o2::utils::Str::tokenize(nameInput2, ':', false, true);
+  if (input2Tokens.size() > 0) {
+    nameInput2 = input2Tokens[0];
+  }
+  if (input2Tokens.size() > 1) {
+    mScaleInput2 = std::stod(input2Tokens[1]);
+  }
```

Comment:
@lhusova you could allow also input1 to have an extra scaling factor. Maybe to be kept in mind for an additional PR